scenarios/jwt_authentication/README.md

```diff
@@ -55,6 +71,13 @@ az role assignment create \
   --scope $res_id
 ```
 
+By assigning these roles to an Azure subscription, it allows that subscription to communicate with any Topic Space within an instance of Event Grid owned by the specified Service Principal (e.g., if these roles are assigned at a subscription level, any Topic Space of an Event Grid under the given subscription could be subscribed/published to).
+
+An alternative to using Azure CLI is the Azure Portal:
+1. Locate the resource group that contains the desired instance of Event Grid.
+2. Navigate to `Access control (IAM)` blade.
+3. Under `Role Assignments`, click `Add`, and assign `Event Grid Topic Spaces Publisher/Subscriber` roles.
+4. Assign the role to the desired Azure account, and click `Review + Assign`.
```

Comment:
do we have an script for this?

Reply:
I believe that it is directly above these directions. I just thought it would be a good idea to add the portal GUI alternative as well.